Automattic/wp-calypso
# Remove hidden site type questions from Jetpack connect flow

Follow on from #34620

#### Changes proposed in this Pull Request

* Remove code for disabled feature "jetpack/connect/site-questions"
* Feature has been disabled for 6 months (since August 2019) 
* See paObgF-k1-p2

#### Testing instructions



* ensure wp-config.php flag for disabling iframe connection is false: `define( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME', false );`
* Head to wp-admin dashboard page. Add ?calypso_env=development to the URL
* Connect via the dashboard widget
* Confirm things are not crashy
